content/data-life-cycle/_index.md

```diff
@@ -22,6 +22,7 @@ The research data life cycle can be divided into several phases as seen in the w
       <div class="card-body">
         {{< data-life-cycle >}}
       </div>
+      The reasearch data life cycle wheel is reused from <a href=https://rdmkit.elixir-europe.org/>RDMKit</a>
```

Comment:
I would have liked this text to be right-aligned and possibly a bit further down but I don't know how to do that so help is appreciated

Reply:
I'm on it 👍

Reply:
RDMkit not RDMKit